cron/notification-for-bekrachtigde-mandataris.ts

```diff
@@ -0,0 +1,89 @@
+import { CronJob } from 'cron';
+
+import { querySudo } from '@lblod/mu-auth-sudo';
+import { getSparqlResults } from '../util/sparql-result';
+import { MANDATARIS_STATUS } from '../util/constants';
+import { sparqlEscapeDateTime } from '../util/mu';
+import { createNotification } from '../util/create-notification';
+
+const NOTIFICATION_CRON_PATTERN =
+  process.env.BESLUIT_CRON_PATTERN || '0 0 * * SUN'; // Every week at 00:00 on monday
```

Comment:
Set this to every week at midnight on Sunday as when we do it every half/day or day it will create a lot of duplicate notifications. How should we check if the notification is still active?

A: check the is-read together with the attached link of the mandataris and the title of a bekrachtigd notification?

Reply:
check notification of subject with linked mandataris else don't send a notification